app/routes/events/components/EventDetail/index.js

```diff
@@ -230,6 +237,32 @@ export default class EventDetail extends Component<Props, State> {
             ),
           }
         : null,
+      activationTimeMoment.isBefore(currentMoment)
+        ? {
+            key: 'Frist for av/påmelding',
+            keyNode: (
+              <Tooltip
+                content={
+                  <span>
+                    Etter påmeldingen stenger er det hverken mulig å melde seg
+                    på eller av arrangementet.
```

Comment:
It might be an idea to add a more thorough explanation of this to the rules page, or? E.g. why it was introduced in the first place and what happens if you still don't show up to the event.

Reply:
Probably a good idea, but that might be a matter we should probably include inter in. Although I think @juniwbjerde talked about updating those rules last semester, not sure what the status is on that. Might write a suggestion for updated rules some time

Reply:
I'll add it to our meeting agenda for next Tuesday!